app/controllers/api/v1/admin/users_controller.rb

```diff
@@ -9,8 +9,13 @@ class UsersController < ApiController
         end
 
         def active_users
+          sort_config = config_sorting(allowed_columns: %w[name roles.name])
+
           # TODO: Change to get active users only
-          users = User.includes(:role).with_provider(current_provider).with_attached_avatar.search(params[:search])
+          users = User.includes(:role)
+                      .with_provider(current_provider)
+                      .with_attached_avatar
+                      .order(sort_config, role_id: :desc)&.search(params[:search])
```

Comment:
By default, should we make it alphabetical by user's name? Just an idea

Reply:
Could be. I think it makes sense to put admins at the top of the list. But it might be less appropriate if a provider has a bunch of custom roles.

Reply:
I think I would let it as is unless I am told that most providers don't use the default role hierarchy of admin-users-guests

Reply:
Yep I like this